sys/include/event/periodic.h

```diff
@@ -75,10 +78,13 @@ void event_periodic_init(event_periodic_t *event_periodic, ztimer_clock_t *clock
  *
  * @param[in]   event_periodic   event_timout context object to use
  * @param[in]   interval         period length for the event
+ * @param[in]   count            time the event should repeat itself,
+ *                               EVENT_PERIODIC_FOREVER to run for ever
```

Comment:
If `count` is 0, will the event be executed once or never?

Reply:
Hmm right... right now it would execute once, but count==0 as well. I would maybe add an assert !=0, would be less confusing than having this be the number of repetitions.

Reply:
Actually it maybe it makes for sense for it to mean `forever` as well and `if(!count) { count = UINT32_MAX; }`

Reply:
I think there are two options:

 - rename `count` -> `repetitions` to make it obvious what is supposed to happen
 - `#define EVENT_PERIODIC_FOREVER 0` instead of `UINT32_MAX`